src/main/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
let’s configure access to h2 database through the console in `application.properties`

Reply:
Ok